lighthouse-core/lib/dependency-graph/network-node.js

```diff
@@ -44,7 +44,7 @@ class NetworkNode extends BaseNode {
   }
 
   /**
-   * @return {?string}
+   * @return {string}
```

Comment:
According to the protocol this shouldn't ever be `null`. Maybe this is just being defensive, but more likely it's just really old so probably hasn't been correct since we moved off `devtools-frontend` for piecing together network records